script/update-test-harness.js

```diff
@@ -3,54 +3,102 @@ const path = require('path')
 const YAML = require('node-yaml')
 
 function writeEnvironmentToFile(os, env) {
-  const environmentVariables = env.map(a => `${a} \\`).join('\n')
```

Comment:
Previously we were just using the environment variables in one command (so you can chain them over newlines with a `\`) but after revisiting this area I realised it was easier to just set them at the top of the script.